app/Jobs/CheckAndUpdateBenchmarkResult.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Helpers\Bitrate;
+use App\Models\Result;
+use Illuminate\Bus\Batchable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Queue\Queueable;
+use Illuminate\Support\Arr;
+
+class CheckAndUpdateBenchmarkResult implements ShouldQueue
+{
+    use Batchable, Queueable;
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct(
+        public Result $result,
+    ) {}
+
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
```

Comment:
You'll want to use the new helper `Benchmark` which will do the bit/byte conversions for you along with assessing if the benchmark passes or not.